testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -7,9 +7,16 @@ import DetailView from './detail-view';
 import ContributorView from './contributor-view';
 import template from '../templates/experiment-page';
 import DisableDialogView from './disable-dialog-view';
+import ExperimentListView from './experiment-list-view';
 import ExperimentTourDialogView from './experiment-tour-dialog-view';
+import TestpilotPromoView from './testpilot-promo-view';
 
-const changeHeaderOn = 127;
+function changeHeaderOn() {
+  const header = document.getElementsByClassName('details-header-wrapper')[0];
+  const content = document.getElementsByClassName('details-content')[0];
```

Comment:
There a reason you are using `getElementsByClassName[0]` over `querySelector`? Is it faster?